CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -183,7 +184,7 @@ shield takes 30 damage and the plasma sword passes through the shield.
 MISCELLANEOUS THROWN WEAPONS
 ==============================
 
-    These weapons work similarly to guns. However your range is determined by 
+	These weapons work similarly to guns. However your range is determined by 

```

Comment:
Why the double indent?

Reply:
ah. Our indent settings are different. We should probably standardize those.

Reply:
I don't normally reject your indents and substitute my own but there were some of each in the same section so I made them all the same.